src/torchpme/potentials/inversepowerlaw.py

```diff
@@ -17,6 +17,30 @@ def gamma(x: torch.Tensor) -> torch.Tensor:
     return torch.exp(gammaln(x))
 
 
+# Auxilary function for stable Fourier transform implementation
+def gammainc_upper_over_powerlaw(exponent, zz):
+    if exponent not in [1, 2, 3, 4, 5, 6]:
+        raise ValueError(f"Unsupported exponent: {exponent}")
+
+    if exponent == 1:
+        return torch.exp(-zz) / zz
+    if exponent == 2:
+        return torch.sqrt(torch.pi / zz) * torch.erfc(torch.sqrt(zz))
+    if exponent == 3:
+        return -torch.expi(-zz)
+    if exponent == 4:
+        return 2 * (
+            torch.exp(-zz) - torch.sqrt(torch.pi * zz) * torch.erfc(torch.sqrt(zz))
+        )
+    if exponent == 5:
+        return torch.exp(-zz) + zz * torch.expi(-zz)
+    if exponent == 6:
+        return (
+            (2 - 4 * zz) * torch.exp(-zz)
+            + 4 * torch.sqrt(torch.pi) * zz**1.5 * torch.erfc(torch.sqrt(zz))
+        ) / 3
+
+
```

Comment:
Can we double-check that everything here is correct? With the new implementation, `test_values_ewald.py` fails for inverse potentials with an exponent of 1, as it does not coincide with the Madelung constant.